cclib/io/ccio.py

```diff
@@ -416,3 +419,55 @@ def _determine_output_format(outputtype, outputdest):
             raise UnknownOutputFormatError(extension)
 
     return outputclass
+
+def path_leaf(path):
+    """
+    Splits the path to give the filename. Works irrespective of '\'
+    or '/' appearing in the path and also with path ending with '/' or '\'.
+
+    Inputs:
+      path - a string path of a logfile.
+    Returns:
+      tail - 'directory/subdirectory/logfilename' will return 'logfilename'.
+      ntpath.basename(head) - 'directory/subdirectory/logfilename/' will return 'logfilename'.
+    """
+    head, tail = ntpath.split(path)
```

Comment:
@berquist So you mean that I should use `os.path.split()` here?

Reply:
Without working it out myself, that sounds right.